src/platform/i386/chal_cpu.h

```diff
@@ -120,7 +121,7 @@ chal_cpu_init(void)
 #if defined(__x86_64__)
 	u32_t low = 0, high = 0;
 
-	chal_cpu_cr4_set(cr4 | CR4_PSE | CR4_PGE);
+	chal_cpu_cr4_set(cr4 | CR4_PSE | CR4_PGE | CR4_PKE);
```

Comment:
Should we check to make sure that MPK exists before setting this? Will it cause systems to crash that don't define the bit? Generally, we can see if it is supported in `cpuid`. This might be a TODO for the future.